client/post-editor/controller.js

```diff
@@ -177,6 +175,15 @@ module.exports = {
 		renderEditor( context, postType );
 	},
 
+	exitPost: function( context, next ) {
+		const postId = getPostID( context );
+		const siteId = getSelectedSiteId( context.store.getState() );
```

Comment:
Maybe it's because https://github.com/Automattic/wp-calypso/blob/master/client/state/initial-state.js#L90 is async. It seems that the context is initialized on the callback of this function https://github.com/Automattic/wp-calypso/blob/master/client/boot/index.js#L260

Is this the only place we have this kind of errors, seems surprising to me